tests/drivers/can/api/src/classic.c

```diff
@@ -504,6 +529,29 @@ ZTEST_USER(can_classic, test_set_bitrate_too_high)
 	zassert_equal(err, 0, "failed to start CAN controller (err %d)", err);
 }
 
+/**
+ * @brief Test setting CAN bitrate to min.
+ */
+ZTEST_USER(can_classic, test_set_bitrate_equal_min)
```

Comment:
Positive test.
API states that can_get_bitrate_min() returns "Minimum supported bitrate in bits/s".
Here, I check that value returned by can_get_bitrate_min() is accepted by can_set_bitrate().

I pointed out that function returns invalid value of 0 bit/s. You explained that value of 0 bit/s has it's meaning.
I think, such behaviour shall be documented.

Reply:
API documentation was extended with "Whether the CAN controller can achieve this bitrate depends on the CAN core clock rate and the minimum CAN timing limits".
Removing this test as it can fail for some combinations of targets and drivers.